### PR TITLE
Feat/contributors page

### PR DESCRIPTION
**Pull request by lepsalex**
* add Acknowledgement page to portal
* queries against Singularity for contributors
* adds Acknowledgement blurb to download info modal

### CODE REVIEW

File: global/hooks/useSingularityData/index.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ */
+
+import { useState } from 'react';
+
+import { getConfig } from '../../config';
+import processStream from '../../utils/processStream';
+import { ContributorsResponse, SingularityErrorResponse } from './types';
+
+const useSingularityData = () => {
+  const { NEXT_PUBLIC_MUSE_API } = getConfig();
```
Comment (jaserud) [inline global/hooks/useSingularityData/index.ts:29]: MUSE? is that right??

File: global/hooks/useSingularityData/index.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  const [awaitingResponse, setAwaitingResponse] = useState(false);
+
+  // For reference: https://singularity.dev.cancogen.cancercollaboratory.org/swagger-ui/
+  const fetchContributors = () => {
+    setAwaitingResponse(true);
+
+    return fetch(`${NEXT_PUBLIC_MUSE_API}contributors`, {
```
Comment (jaserud) [inline global/hooks/useSingularityData/index.ts:36]: Just a suggestion, might be better to use urlJoin then just string template so it doesn't rely on configured url to have the ending `/`

File: next.config.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ module.exports = withCSS({
     NEXT_PUBLIC_LAB_NAME: process.env.NEXT_PUBLIC_LAB_NAME,
     NEXT_PUBLIC_LOGO_FILENAME: process.env.NEXT_PUBLIC_LOGO_FILENAME,
     NEXT_PUBLIC_MUSE_API: process.env.NEXT_PUBLIC_MUSE_API_URL,
+    NEXT_PUBLIC_MUSE_API: process.env.NEXT_PUBLIC_SINGULARITY_API_URL,
```
Comment (jaserud) [inline next.config.js:23]: looks like some copy pasta error

File: pages/acknowledgements/index.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+const TeamPage = createPage({
+  getInitialProps: async ({ query, egoJwt }) => {
+    return { query, egoJwt };
+  },
+  isPublic: true,
+})(() => {
+  return <Team />;
+});
+
+export default TeamPage;
```
Comment (jaserud) [inline pages/acknowledgements/index.tsx:35]: `Acknowledgements` and `AcknowledgementsPage` might be better names lol

**Review by jaserud — CHANGES_REQUESTED** [review]
need to fix some copy pasta configs

File: global/hooks/useSingularityData/index.ts (hunk 1 above)
Comment (lepsalex) [reply global/hooks/useSingularityData/index.ts:29]: no it is not

**Review by jaserud — APPROVED** [review]

**Pull request merged**